pkg/network/node/ovscontroller.go

```diff
@@ -267,6 +268,17 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 
 	// Table 10: VXLAN ingress filtering; filled in by AddHostSubnetRules()
 	// eg, "table=10, priority=100, tun_src=${remote_node_ip}, actions=goto_table:30"
+	// Swap overlay dst MAC for ingress OVN-K hybrid overlay traffic
+	if oc.localMAC == "" {
+		return fmt.Errorf("failed to find the MAC address for local IP %s", oc.localIP)
+	}
+	if err := oc.ensureTunMAC(); err != nil {
+		return err
+	}
+	// set the dst MAC for traffic from OVN-K nodes to local node
+	otx.AddFlow("table=10, priority=210, ip, nw_dst=%s, dl_dst=%s, actions=set_field:%s->eth_dst, resubmit:10", localSubnetGateway, oc.localMAC, oc.tunMAC)
+	// set the dst MAC for traffic from OVN-K nodes to local pods
+	otx.AddFlow("table=10, priority=200, ip, nw_dst=%s, dl_dst=%s, actions=move:nw_dst->dl_dst[0..31], set_field:0a:58:00:00:00:00/ff:ff:00:00:00:00->eth_dst, resubmit:10", localSubnetCIDR, oc.localMAC)
```

Comment:
use `eth_dst` everywhere rather than saying `dl_dst` (which is an alias) in some places.

(also, same question on the `resubmit`)

Reply:
fixed